### ISSUE TITLE
Rename mitxgraders/helpers/mitmath to mitxgraders/helpers/calc

### ISSUE THREAD

**Issue by ChristopherChudzicki** [mitodl/mitx-grading-library#87]
- mitxgraders.helpers.mitmath is silly, it has two instances of `'mit'`
- and it's long
- Origianlly I wanted it to just be mitxgraders.helpers.math, but that caused naming conflicts in some doctests

Suggestion: use `mitxgraders.helpers.calc` instead. This would mirror what edX does. Their calc module has structure
```
calc
|--__init__.py
|--calc.py
|--functions.py
|--preview.py
```

**Comment by jolyonb** [comment]
Fine by me.

**Comment by jolyonb** [comment]
Resolved by #92.
